Sources/CSQLite3/shim.c

```diff
@@ -0,0 +1,2 @@
+
+unsigned int _;
```

Comment:
?

Reply:
C formally does not allow an empty file, at least one declaration must be present.  This declaration both ensures that the source file is ANSI C compliant and silences a warning from ld64 that no debug information is associated with the translation unit.